iOSMoviesApp/CustomTableViewCells/CastCollectionViewCell/CastCollectionViewCell.swift

```diff
@@ -0,0 +1,81 @@
+//
+//  CastCollectionViewCell.swift
+//  iOSMoviesApp
+//
+//  Created by Yash Agrawal on 11/07/25.
+//
+
+import UIKit
+
+
+protocol CastCollectionViewCellDelegate: AnyObject {
+    func updateProfilePic(with imageData : Data, cacheKey: NSString)
+    func updateProfilePicFromCache(with image: UIImage)
+    func updateProfilePicFromPlaceholder(named: String)
+}
+
+class CastCollectionViewCell: UICollectionViewCell {
+    
+    @IBOutlet weak var realName: UILabel!
+    @IBOutlet weak var fictionalName: UILabel!
+    @IBOutlet weak var castImage: UIImageView!
+    private var castVM : CastViewModel!
```

Comment:
Try using `Optional` here, if there is any case where the `ViewModel` not be initialized. Accessing an uninitialized `ViewModel` can lead to a crash.
 
```suggestion
    private var castVM : CastViewModel?
```